homeassistant/components/peblar/update.py

```diff
@@ -41,8 +42,9 @@ class PeblarUpdateEntityDescription(UpdateEntityDescription):
     PeblarUpdateEntityDescription(
         key="customization",
         translation_key="customization",
-        installed_fn=lambda x: x.current.customization,
         available_fn=lambda x: x.available.customization,
+        has_fn=lambda x: x.current.customization is not None,
```

Comment:
It seems `has_fn` is not implemented in `async_setup_entry`. You plan that to do that in a follow-up or did you miss that accidentally?

Reply:
🤦  That is a miss